storage/src/tests/distributor/distributor_stripe_test.cpp

```diff
@@ -144,10 +144,8 @@ struct DistributorStripeTest : Test, DistributorStripeTestUtil {
         });
     }
 
-    void configure_update_fast_path_restart_enabled(bool enabled) {
-        configure_stripe_with([&](auto& builder) {
-            builder.restartWithFastUpdatePathIfAllGetTimestampsAreConsistent = enabled;
-        });
+    void configure_update_fast_path_restart_enabled() {
+        configure_stripe_with([&](auto& builder) { (void) builder; });
```

Comment:
Consider adding a small comment on why this is currently a no-op, lest we forget many moons from now

Reply:
Good catch, actually unused.